### DISCUSSION TITLE
IIS Hosted Application configuration with Skyapm agent

### DISCUSSION THREAD

**Discussion by AjinThankachan** [SkyAPM/SkyAPM-dotnet#504] [Q&A]
Hi,
We are having application which is hosted on IIS. But the problem is when ever running the script it was not able to identify the project using the command(dotnet add package SkyAPM.Agent.AspNetCore). The hosted application doesn't have any .csproj or startup.cs or Program.cs file. So how can i configure the application with Skyapm. Can you please help me to do it. It's a standalone .net application it seems.
Thanks in advance

**Comment by liuhaoyang** [comment]
Please make sure your project type is ASP.Net Core or ASP.Net

**Reply by StevenWangGitHub** [reply]
ASP.Net
